### PR TITLE
Aliases option

### PR DESCRIPTION
**Pull request by bradrisse**
Gives the user the ability to have an object array of aliases for each icon. The example in the test I used was using "git" in addition to "github". This satisfies issue #41

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -90,6 +91,16 @@ function iconfontCSS(config) {
 			codePoint: currentCodePoint
 		});
 
+		if (config.aliases[fileName]) {
+			_.each(config.aliases[fileName], function(_alias) {
+				glyphMap.push({
+					fileName: _alias,
+					codePoint: currentCodePoint,
+					aliasRoot: fileName // used for less and scss
```
Comment (backflip) [inline index.js]: Tiny detail: Took me some time to figure out the reason for this, would you mind renaming it to `originalFileName` or something?

**Comment by backflip** [comment]
@bradrisse, thanks for this well-prepared PR! Only one comment above.

**Comment by bradrisse** [comment]
@backflip aliasRoot has been changes and conflicts have been resolved

**Comment by backflip** [comment]
@bradrisse, thanks a lot, published as 2.3.0

**Pull request merged**